decryptor/base/decryptor.go

```diff
@@ -97,7 +97,8 @@ type DataDecryptor interface {
 	// acrastruct using secure message
 	// return decrypted data or data as is if fail
 	// db specific
-	ReadSymmetricKey([]*keys.PrivateKey, io.Reader) ([]byte, []byte, error)
+	ReadSymmetricKey(*keys.PrivateKey, io.Reader) ([]byte, []byte, error)
+	ReadSymmetricKeyRotated([]*keys.PrivateKey, io.Reader) ([]byte, []byte, error)
```

Comment:
adding this method doesn't touch acra-ee? it wasn't used directly anywhere? if so, then lgtm

Reply:
> adding this method doesn't touch acra-ee?

It _does_ affect Acra EE. I mean, it will once this is merged into master *and* Acra EE is updated to use that commit.

Because we add new methods to interface that Acra EE reimplements from scratch in its own code (instead of inheriting Acra CE's implementation), Acra EE will need to be updated to implement new methods too. I plan to prepare a *keystore-v2* branch in Acra EE repo with relevant changes before CE's keystore-v2 lands into master. Then this update can be coordinated.